### ISSUE TITLE
OneHotEncoding: Train/Test data set

### ISSUE THREAD

**Issue by choshin84** [choshin84/learning_memo#25]
# Tweet summary
Better use sklearn OneHotEncoder fit/transform to ensure test data set will be OnHotEncoded by training data set.
<watch-out> when using "drop first" option to eliminate co-linearity, ignore new label option cannot be used

# Useful link
https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.OneHotEncoder.html

**Comment by choshin84** [comment]
More handy with category name in OneHotEncoding output dataframe
http://contrib.scikit-learn.org/categorical-encoding/onehot.html
